wger/exercises/migrations/0013_auto_20201202_0043.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.1.3 on 2020-12-01 23:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('exercises', '0012_auto_20201201_1258'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='ExerciseVariation',
+        ),
+        migrations.AlterField(
+            model_name='exercise',
+            name='variations',
+            field=models.ManyToManyField(blank=True, related_name='_exercise_variations_+', to='exercises.Exercise', verbose_name='Variations'),
+        ),
+    ]
```

Comment:
That's a lot of migrations 😅

You can go back to the last one with `python3 manage.py migrate exercises 0005`, delete the new ones and then generate a single one with all the changes